tests/test_minimal/test_tools/test_backend_and_dataset_configuration/test_helpers/test_configure_backend_defaults.py

```diff
@@ -58,3 +56,33 @@ def test_simple_time_series(
             assert written_data.compression == "gzip"
         elif backend == "zarr":
             assert written_data.compressor == numcodecs.GZip(level=1)
+
+
+@pytest.mark.parametrize("backend", ["hdf5", "zarr"])
+def test_simple_dynamic_table(tmpdir: Path, backend: Literal["hdf5", "zarr"]):
+    data = np.zeros(shape=(30_000 * 5, 384), dtype="int16")
+
+    nwbfile = mock_NWBFile()
+    dynamic_table = DynamicTable(
+        name="TestDynamicTable", description="", columns=[VectorData(name="TestColumn", description="", data=data)]
+    )
+    nwbfile.add_acquisition(dynamic_table)
+
+    backend_configuration = get_default_backend_configuration(nwbfile=nwbfile, backend=backend)
+    dataset_configuration = backend_configuration.dataset_configurations["acquisition/TestDynamicTable/TestColumn/data"]
+    configure_backend(nwbfile=nwbfile, backend_configuration=backend_configuration)
+
+    nwbfile_path = str(tmpdir / f"test_configure_defaults_dynamic_table.nwb.{backend}")
+    with BACKEND_NWB_IO[backend](path=nwbfile_path, mode="w") as io:
```

Comment:
I would also avoid inlining the IO class here. I think it makes the code harder to read but is marginal.

Reply:
Sure